crates/shared/src/order_validation.rs

```diff
@@ -1648,64 +1654,40 @@ mod tests {
         // *** SELL ORDERS ***
         // at market price
         assert!(!is_order_outside_market_price(
-            &OrderData {
-                sell_amount: 100.into(),
-                buy_amount: 100.into(),
-                kind: OrderKind::Sell,
-                ..Default::default()
-            },
+            &U256::from_dec_str("100").unwrap(),
+            &U256::from_dec_str("100").unwrap(),
```

Comment:
nit: The old syntax still works and is a bit nicer to read IMO.
```suggestion
            &100.into(),
            &100.into(),
```